core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -518,4 +518,13 @@ public enum SqlConformanceEnum implements SqlConformance {
       return false;
     }
   }
+
+  @Override public boolean isNonStrictGroupBy() {
+    switch (this) {
+    case BABEL:
+      return true;
```

Comment:
Whether isNonStrictGroupBy should be enabled by default for MySQL. This mode is very common in MySQL.